jsinterop-ts-defs-processor/src/main/java/com/vertispan/tsdefs/model/TsClass.java

```diff
@@ -31,9 +33,9 @@ public class TsClass implements IsType {
   private TsClass superClass;
   private List<TsInterface> interfaces = new ArrayList<>();
   private List<TsModifier> modifiers = new ArrayList<>();
-  private List<TsProperty> properties = new ArrayList<>();
+  private Set<TsProperty> properties = new HashSet<>();
```

Comment:
should order never matter? would it be safer to use LinkedHashSet as the implementation here?

Reply:
Hmm, in what case the order of the fields in the TS class should matter? nothing came to my mind. :thinking:

Reply:
I tend to think that the order in java had some meaning, and that it might make sense to reflect that in TS? But yes, there is no intrinsic meaning, only the order the developer wrote down.